Stronger/Account/AccountSetupHeader.swift

```diff
@@ -22,8 +22,8 @@ struct AccountSetupHeader: View {
                 .bold()
                 .padding(.bottom)
                 .padding(.top, 30)
-            Text("ACCOUNT_SUBTITLE")
-                .padding(.bottom, 8)
+            Text("Please sign-in to continue your Stronger journey")
```

Comment:
Great title 🚀 

Please be sure that you remove any stale localization strings from the Strings catalog. You can learn more about localization here and I would highly recommend watching the video to completely understand the localization process that we use in Spezi-based applications: https://developer.apple.com/wwdc23/10155